src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorDropHandlerProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Editor.DragDrop;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    [Export(typeof(IDropHandlerProvider))]
+    [ContentType(RazorLSPConstants.RazorLSPContentTypeName)]
+    [DropFormat(RazorLSPConstants.VSProjectItemsIdentifier)]
+    [Name(nameof(RazorDropHandlerProvider))]
+    [Order(Before = "LanguageServiceTextDropHandler")]
```

Comment:
Where did this string come from?

Reply:
From WTE's code. I'm not quite sure the implication to be honest though.